### PR TITLE
if webview show app name

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref #876 
In regular what's new screen there is no text.

fileo4uvrk

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
👍 fine by me, please review & merge @mario

**Comment by mario** [comment]
@AndyScherzinger already reviewed, see my comment - I don't understand why the text was hardcoded to white color.

**Comment by mario** [comment]
@tobiasKaminsky comments made.

**Comment by mario** [comment]
@AndyScherzinger you can check my comment here as well.

File: src/main/res/layout/whats_new_activity.xml [hunk 1]
```diff
@@ -29,12 +29,15 @@
...
+        android:layout_marginLeft="10dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="5dp"
         android:layout_weight="6"
-        android:gravity="center"
+        android:gravity="center_vertical"
         android:text="@string/placeholder_sentence"
         android:textAppearance="?android:attr/textAppearanceLarge"
-        android:textColor="@color/login_text_hint_color"/>
+        android:textColor="@color/white"/>
```
Comment (tobiasKaminsky) [inline src/main/res/layout/whats_new_activity.xml]: Valid point 👍 
So better use "primary_button_text_color"?

Comment (mario) [inline src/main/res/layout/whats_new_activity.xml]: Yes.

**Comment by tobiasKaminsky** [comment]
Changes done.

**Comment by mario** [comment]
👍

**Pull request merged**